### DISCUSSION TITLE
esx_advancedgarage

### DISCUSSION THREAD

**Discussion by Ansaca17** [community/community#56727] [Programming Help]
### Select Topic Area

Question

### Body

Hello, can someone help me please.
How can I do so that the vehicle when stored in the garage can only be taken out in that same garage and not in the others?

The script this:
https://github.com/HumanTree92/VENT_ESX_Scripts/tree/main/esx_advancedgarage

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Ansaca17 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
